aimmo-game/tests/functional/test_damage_pickups_and_effects.py

```diff
@@ -0,0 +1,55 @@
+from unittest import TestCase
+
+from mock_world import MockWorld
+
+from simulation.location import Location
+from simulation.pickups import DamageBoostPickup
+
+
+class TestDamagePickupsAndEffects(TestCase):
+    def setUp(self):
+        """
+        Mock a game for each test individually. MockWorld() will set up a:
+        avatar manager, game state, turn manager and a map generator.
+        """
+        self.game = MockWorld()
+        self.game.game_state.add_avatar(1, None, Location(0, 0))
+        self.cell = self.game.game_state.world_map.get_cell(Location(1, 0))
+
+    def test_damage_boost_pickup_can_be_picked_up_default(self):
+        pickup_created = DamageBoostPickup(self.cell)
+        self.cell.pickup = pickup_created
+
+        self.game.turn_manager._run_single_turn()
+
+        self.assertEqual(self.cell.avatar, self.game.avatar_manager.get_avatar(1))
+        self.assertEqual(len(self.game.avatar_manager.get_avatar(1).effects), 1)
+        damage_boost_effect = self.game.avatar_manager.get_avatar(1).effects.pop()
+        self.assertTrue(isinstance(damage_boost_effect, pickup_created.EFFECT))
+
+    def test_damage_boost_pickup_can_be_picked_up_custom(self):
+        pickup_created = DamageBoostPickup(self.cell, 15)
```

Comment:
we need to make sure in the code that the custom figure is different from the default one I think?

Reply:
In `DamageBoostPickup`? Why? If the user provides 5, and the default is 5, it makes no difference to the functionality

Reply:
that's because the parameter is hardcoded... if it was a setting somewhere now.......
Full disclaimer: ok you can check the default value https://stackoverflow.com/questions/12627118/get-a-function-arguments-default-value